### PR TITLE
Fix README.md Text changes

### PR DESCRIPTION
**Pull request by dixitadusara**

### CODE REVIEW

**Comment by dixitadusara** [comment]
@GaryJones,

I have found another text changes. Please check this PR.

File: README.md [hunk 1]
```diff
@@ -58,13 +58,13 @@ Running this command will:
 3. Register WordPress standards in PHP_CodeSniffer configuration.
 4. Make `phpcs` command available from `wpcs/vendor/bin`.
 
-For convenience of using `phpcs` as global command you might want to add path to `wpcs/vendor/bin` directory to a `PATH` environment of your operating system.
+For the convenience of using `phpcs` as global command, you might want to add path to `wpcs/vendor/bin` directory to a `PATH` environment of your operating system.
```
Comment (GaryJones) [inline README.md]: Needs "a" before "global".

File: README.md [hunk 2]
```diff
@@ -58,13 +58,13 @@ Running this command will:
...
 
 ### Standalone
 
 1. Install PHP_CodeSniffer by following its [installation instructions](https://github.com/squizlabs/PHP_CodeSniffer#installation) (via Composer, PEAR, or Git checkout).
 
-   Do ensure, if for example you're using [VVV](https://github.com/Varying-Vagrant-Vagrants/VVV), that PHP_CodeSniffer's version matches our [requirements](#requirements).
+   Do ensure, if for example, you're using [VVV](https://github.com/Varying-Vagrant-Vagrants/VVV), that PHP_CodeSniffer's version matches our [requirements](#requirements).
```
Comment (GaryJones) [inline README.md]: Do ensure that PHP_CodeSniffer's version matches our [requirements](#requirements), if, for example, you're using [VVV](https://github.com/Varying-Vagrant-Vagrants/VVV).

**Comment by dixitadusara** [comment]
@GaryJones,

I have changed as per your suggestion.

**Comment by GaryJones** [comment]
Thanks again @dixitadusara :-)

**Pull request merged**